src/common/chat-participants/vscode.proposed.chatVariableResolver.d.ts

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
for png above - i believe we should be using the new colorful logo

Reply:
I'll check with Manajit on this